packages/jest-config/src/utils.js

```diff
@@ -0,0 +1,126 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @flow
+ */
+
+'use strict';
+
+import type {Path} from 'types/Config';
+
+const {ValidationError} = require('jest-validate');
+const Resolver = require('jest-resolve');
+const path = require('path');
+const chalk = require('chalk');
+
+const throwValidationError = (message: string) => {
+  throw new ValidationError(
+    null,
+    chalk.white(`\n\n  ${message}`)
```

Comment:
chalk.reset here too.

Shouldn't this be done inside `ValidationError`? Why do you need to do this from the outside everywhere?